src/source/source_simple_fuse.ts

```diff
@@ -0,0 +1,23 @@
+import { z } from "zod"
+import {
+  source_component_base,
+  type SourceComponentBase,
+} from "./base/source_component_base"
+import { expectTypesMatch } from "src/utils/expect-types-match"
+
+export const source_simple_fuse = source_component_base.extend({
+  ftype: z.literal("simple_fuse"),
+  current_rating: z.union([z.number(), z.string()]),
+  voltage_rating: z.union([z.number(), z.string()]).optional(),
+})
+
+export interface SourceSimpleFuse extends SourceComponentBase {
+  ftype: "simple_fuse"
+  current_rating: number | string
+  voltage_rating?: number | string
```

Comment:
this shouldn't be a string, or if it is the unit is unclear. circuit_json is more exact and less forgiving than props. You have to either 1) use an enum or 2) use a number. I believe you can do `voltage_rating_volts: number` with `.describe("Voltage rating e.g. +-5V = 5")`, for current rating you can do `current_rating_amps: number` `.describe("Nominal current in amps fuse is rated for")`